esp-hal/src/lcd_cam/lcd/mod.rs

```diff
@@ -14,36 +14,53 @@ use crate::{peripheral::PeripheralRef, peripherals::LCD_CAM};
 
 pub mod i8080;
 
+/// Represents an LCD interface peripheral.
```

Comment:
You call the camera and i8080 an interface, not an interface peripheral. We might want to be consistent with those.